api/app/src/main/kotlin/packit/service/PacketService.kt

```diff
@@ -44,14 +45,33 @@ class BasePacketService(
     private val outpackServerClient: OutpackServer
 ) : PacketService
 {
+    // Return the display name for a packet if its custom metadata schema conforms to the orderly schema and contains
+    // a display name.
+    // Also check for 'display name' keys that may exist in non-orderly outpack custom schemas.
+    // Falls back to name if no display name.
+    private fun getDisplayNameForPacket(packet: OutpackMetadata): String {
```

Comment:
also didnt see unit tests for this method..... change to internal to be able to test and expose

Reply:
It's tested by the tests of importPackets. I think private functions are allowed, and that they shouldn't be separately unit tested.

Reply:
Because I should test the public interface ('guaranteeing' some level of stability for it), not the implementation details (which should be allowed to be changeable / not stable)

Reply:
```kt
val expectedDisplayNames = mapOf(
            "20240101-090000-4321gaga" to "test", // This packet has no display name, so should fall back to its name.
            "20190203-120000-1234dada" to "test name (latest display name)",
            "20190403-120000-1234dfdf" to "test2 name"
        )
```

That's the bit that tests the 'fall back' condition.

I don't bother testing for the non-orderly (outpack) conditions that speculatively check for keys called 'display', but could be persuaded to, just makes the test suite more verbose.